ambry-clustermap/src/main/java/com.github.ambry.clustermap/StaticClusterAgentsFactory.java

```diff
@@ -88,6 +89,21 @@ public void close() {
         public boolean setReplicaSealedState(ReplicaId replicaId, boolean isSealed) {
           return false;
         }
+
+        @Override
+        public boolean setReplicaStoppedState(List<ReplicaId> replicaIds, boolean isStopped) {
+          return false;
+        }
+
+        @Override
+        public List<String> getSealedReplicas() {
+          return new ArrayList<>();
```

Comment:
minor: you can return `Collections.emptyList()` instead

Reply:
fixed. Thanks!